src/main/java/org/hawkular/client/android/activity/AuthorizationActivity.java

```diff
@@ -98,7 +87,11 @@ public void setUpAuthorization() {
             return;
         }
 
-        setUpBackendAuthorization();
+        Intent login_activity = new Intent(getApplicationContext(), LoginActivity.class);
```

Comment:
There is the `Intents` factory-like class specifically for this. And please use camel case for variables declaration.